src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -144,6 +144,25 @@ module Plonk_constraint = struct
       | EC_endoscale of
           { state : 'v Endoscale_round.t array; xs : 'v; ys : 'v; n_acc : 'v }
       | EC_endoscalar of { state : 'v Endoscale_scalar_round.t array }
+      | RangeCheck0 of
+          { v : 'v (* Value to constrain to 88-bits *)
+          ; vp0 : 'v (* MSBs *)
+          ; vp1 : 'v (* vpX are 12-bit plookup chunks *)
+          ; vp2 : 'v
+          ; vp3 : 'v
+          ; vp4 : 'v
+          ; vp5 : 'v
+          ; vc0 : 'v (* vcX are 2-bit crumbs *)
+          ; vc1 : 'v
+          ; vc2 : 'v
+          ; vc3 : 'v
+          ; vc4 : 'v
+          ; vc5 : 'v
+          ; vc6 : 'v
+          ; vc7 : 'v (* LSBs *)
+          ; coeff : 'f
```

Comment:
Nit: it might make more sense to rename this to `compact: bool` and then generate the coefficient in the logic with
```
let coeff = if compact then Fp.one else Fp.zero
```

Reply:
yes - thank you, good suggestion